hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/pipeline/ClosePipelineSubcommand.java

```diff
@@ -38,14 +40,59 @@
   @CommandLine.ParentCommand
   private PipelineCommands parent;
 
-  @CommandLine.Parameters(description = "ID of the pipeline to close")
+  @CommandLine.Parameters(description = "ID of the pipeline to close,"
+      + "'ALL' means all pipeline")
   private String pipelineId;
 
+  @CommandLine.Option(names = {"-ffc", "--filterByFactor"},
```

Comment:
Can you add some acceptance tests for this as you've done in the description?

Reply:
Is there some examples for me to reference? I'd like to follow our test framework.

Reply:
I've got some help from @timmylicheng